packages/utilities/src/utilities/StickyHeader/StickyHeader.js

```diff
@@ -10,6 +10,10 @@ import root from 'window-or-global';
 import settings from '../settings/settings.js';
 
 const { prefix, stablePrefix: c4dPrefix } = settings;
+
+const ddsPrefix = 'dds';
+const prefixV1 = 'bx';
```

Comment:
The name `prefixV1` reads as ambiguous when it's all by itself below. Should it just be `bxPrefix` like the other name above?